test_runner/build.gradle.kts

```diff
@@ -237,3 +236,34 @@ tasks.create("updateFlank", Exec::class.java) {
     description = "Update flank jar"
     commandLine = listOf("./bash/update_flank.sh")
 }
+
+// begin --- ASCII doc generation ---
+val generateManpageAsciiDoc by tasks.registering(JavaExec::class) {
+    dependsOn(tasks.classes)
+    classpath(
+        configurations.compile,
+        configurations.annotationProcessor,
+        sourceSets["main"].runtimeClasspath
+    )
+    group = "Documentation"
+    description = "Generate AsciiDoc manpage"
+    main = "picocli.codegen.docgen.manpage.ManPageGenerator"
+    args = listOf(
+        application.mainClassName,
+        "--outdir=${project.buildDir}/generated-picocli-docs",
+        "-v"
+    )
+}
+
+tasks.asciidoctor {
+    dependsOn(generateManpageAsciiDoc)
+    setBackends("html5")
+    sourceDir = file("${project.buildDir}/generated-picocli-docs")
+    outputDir = file("${project.rootDir}/docs")
+    logDocuments = true
+}
+
+tasks.assemble {
+    dependsOn(tasks.asciidoctor)
+}
+// end --- ASCII doc generation ---
```

Comment:
![image](https://user-images.githubusercontent.com/1173057/80181334-7fee7780-85b9-11ea-81bd-ced0c0808ba8.png)
let's end with a newline :)

Reply:
Ahh sorry. No more missing newlines, finally I checked `Ensure line feed at file end on save` in IDE.